Core/include/Acts/SpacePointFormation/SpacePointBuilderOption.h

```diff
@@ -0,0 +1,21 @@
+// This file is part of the Acts project.
```

Comment:
Final nitpick comment:

We usually try to have file names and class names corresponding,
so I would suggest to call both file and struct `SpacePointBuilderOptions`.

Reply:
Fixed.